javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/index.html

```diff
@@ -0,0 +1,180 @@
+                    fill="#F5F6F7"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="playlist">
+        <div class="playlist-bar">
+          <div class="parallel-lines">
+            <div></div>
+            <div></div>
+          </div>
+          <h2 class="playlist-title" id="playlist">Playlist</h2>
+          <div class="parallel-lines">
+            <div></div>
+            <div></div>
+          </div>
+          <div>
+            <button
+              class="playlist-close"
+              aria-label="Playlist"
+              aria-expanded="true"
+            >
+              X
+            </button>
+          </div>
```

Comment:
It doesn't seem like the hamburger at the top has any functionality. Is it possible to move and change it function to alternate with the X button such that when a user clinks the button the playlist closes and the sign changes into the hamburger which in return when clicked can display the playlist?

Reply:
The top hamburger look-a-line, was just for decoration. I have remove this top 'hamburger' svg and updated the closing playlist button. This new arrow icon is accessible/UX and more intuitive for campers to know what the button/icon is for, a collapsable button.

Thanks for the observation @larymak